app/utils/goalFun.js

```diff
@@ -0,0 +1,13 @@
+function render(route, parent, inModal){
```

Comment:
Since the function is called render, and it renders goals, I would rename `goalFun` to `renderGoal` :smiley: